applications/StructuralMechanicsApplication/python_scripts/structural_response_function_factory.py

```diff
@@ -15,18 +15,23 @@ def CreateResponseFunction(response_id, response_settings, model):
     elif response_type == "eigenfrequency":
         return structural_response.EigenFrequencyResponseFunction(response_id, response_settings, model)
 
-    elif response_type == "adjoint_nodal_displacement":
+    elif response_type == "adjoint_nodal_displacement" or\
```

Comment:
```suggestion
    elif response_type in ["adjoint_nodal_displacement", "adjoint_linear_strain_energy", ...]:
```
this would be another alternative of doing this